src/components/Articles.jsx

```diff
@@ -1,9 +1,53 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import Article from "./article";
 
 export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    fetch("https://news-3fmi.onrender.com/api/articles")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        setArticles(data.articles);
+      });
+  }, []);
+
   return (
-    <div>
-      <h2>Articles</h2>
-    </div>
+    <>
+      <div className="card-container">
+        {articles.map(
+          ({
+            title,
+            topic,
+            author,
+            article_img_url,
+            votes,
+            article_id,
+            comment_count,
+          }) => {
+            return (
+              <div key={article_id} className="card">
+                <div className="card-content">
+                  <div className="image-container">
+                    <img src={article_img_url} alt={title} />
+                  </div>
+                  <div className="text-content">
+                    <h2>{title}</h2>
+
+                    <p>{topic}</p>
```

Comment:
You could add **Topic:** here to make it consistent with the rest of the design 👍